backend/api/Areas/Leases/Controllers/LeaseController.cs

```diff
@@ -72,6 +72,23 @@ public IActionResult AddLease(Models.Lease.LeaseModel leaseModel, bool userOverr
 
             return new JsonResult(_mapper.Map<Models.Lease.LeaseModel>(lease));
         }
+
+        /// <summary>
+        /// Update the specified lease. Allows the user to override the normal restriction on adding properties already associated to a lease.
+        /// </summary>
+        /// <returns></returns>
+        [HttpPut("{leaseId:long}/tenants")]
```

Comment:
nit: Line 47 says `[HttpGet("{id:int}")]` - this one says the LeaseId is a long. Not likely that will we will reach the upper bound of int, but would prefer if we choose one or the other to be consistent